Basic files

kusiLaw opened the pull request:
## Add Basic File
this commit adds the basic (HTML, CSS, readme) files to the project.

kusiLaw committed:
update readme file

kusiLaw committed:
Add git files

it adds .gitignore and html/css linter file

kusiLaw committed:
Add index.html files

The index file sets the page header

kusiLaw committed:
Add css file

this commit style the page header

kusiLaw committed:
add newline to html and css file

kusiLaw committed:
fix linter issue

victorokoroji commented on README.md:
```diff
@@ -1,2 +1,18 @@
 # Hello-Microverse
-Microverse project 
+Microverse project
+
+### Built With
+Html<br />
+CSS
+
+### Author
+<a href="https://github.com/kusiLaw">Github</a> \
+<a href="https://twitter.com/kusilaw">Twitter</a> \
+<a href="https://www.linkedin.com/in/lawrence-kusi-55a662104">Linkedin</a>
+
+### Acknowledgments
+Thanks to Microverse
+
+### License
+This Project is MIT licensed
```
- You did a great job in your readme structure 👍 . Kindly add the [Getting started section ](https://github.com/microverseinc/readme-template/blob/master/README.md) by telling the user how to have your project running locally on their machine. Example:
## Getting Started
To get a local copy up and running follow these simple example steps.

- How to clone the repo, how to move into the cloned repo,
- Prerequisite: e.g vscode editor, a web browser(chrome, etc).

This will make your readme look more professional 😄

victorokoroji replied:
I would suggest you install linters again and see what you will get.

victorokoroji commented on README.md:
```diff
@@ -1,2 +1,18 @@
+Thanks to Microverse
+
+### License
+This Project is MIT licensed
```
- kindly update the MIT license link. You can use the MIT from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add it to the root directory of your project to make your readme appear more professional.

victorokoroji reviewed (changes requested):
Hi @kusiLaw, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 


## Required Changes ♻️

- [ ] I noticed that the test.md file is missing in your project, Kindly add it.
- [ ] Kingly add the [HTML and CSS linters](https://github.com/microverseinc/linters-config) as specified in the requirements. Adding these linter files is to make sure your HTML and CSS code is clean and well organized 💯 
- [ ] Please remember to add the `test.md ` file and `node_modules` folder to the `.gitignore` file to avoid adding unnecessary files to your repository 😄 

_Check the comments under the review._
<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kusiLaw committed:
fix basic files issues

it adds the HTML and CSS linters and License
it also update readme and .gitignore files

kusiLaw commented:
upon installing the command in HTML and CSS linters, this is the issue am having

kusiLaw committed:
reinstall linters

victorokoroji commented:
i can see that your linters is constantly failing. I would have loved to help but i am currently busy. you can go through [FAQ section](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-) to seek help from a senior code reviewer. Thanks

kusiLaw commented:
I have solved the linter issue. but the local repository Head has one step back to this here on GitHub. so I cant push it. 
local commits points to **f7379bc8dfc21f5bda7243563e2d5342a441a1d6** while 
github commits points to **90fae1b09b4136fbc1873f41e6fb39f2df71a86b**

![Screenshot from 2022-06-30 13-02-52](https://user-images.githubusercontent.com/52538840/176689093-8dc58831-0417-4ce3-bd28-e770fe1be706.png)
